### ISSUE TITLE
something is possibly wrong with addElementToBeginningOfArray and addElementToEndOfArray tests

### ISSUE THREAD

**Issue by bboroda** [learn-co-students/javascript-arrays-bootcamp-prep-000#103]
I used the same implementation for both addElementToBeginningOfArray and destructivelyAddElementToBeginningOfArray functions and I passed the tests (in other words, the test didn't catch that I was modifying the array with unshift operation).

Original implementation that passed arrays-test.js:

function addElementToBeginningOfArray(array, element) {
  array.unshift(element);
  return array;
}

function destructivelyAddElementToBeginningOfArray(array, element) {
  array.unshift(element);
  return array;
}

**Comment by albinovulcan** [comment]
Yeah, there are bugs in those two tests, [line 29](https://github.com/learn-co-students/javascript-arrays-bootcamp-prep-000/blob/master/test/arrays-test.js#L29) and [line 57](https://github.com/learn-co-students/javascript-arrays-bootcamp-prep-000/blob/master/test/arrays-test.js#L57).

The `expect(array).to.eql(array)` will always pass - the implementation should be `expect(array).to.eql([1])` in both tests.

**Comment by albinovulcan** [comment]
Heh, I have no idea if a pull request is appropriate here to fix.

**Comment by koko** [comment]
I fixed them in my pull request, if it is appropriate.
